frontend/src/queries/useBakerQuery.ts

```diff
@@ -80,6 +80,11 @@ const BakerQuery = gql<BakerResponse>`
 							rank
 							total
 						}
+						paydayCommissionRates {
+							transactionCommission
+							finalizationCommission
```

Comment:
Finalization commission goes away. It is 0 and should not be shown.

Reply:
Removed (was not used anymore)